README.md

```diff
@@ -1,10 +1,24 @@
 # Component Finder
 
-A tool that clones all `-frontend` service repositories and any white-listed service repositories, searches their `*.html` files for a `<searchString>` and provides a list of files that contain said `<searchString>`.
+A tool to search repositories for a string.
 
 [![Build Status](https://travis-ci.org/hmrc/component-finder.svg?branch=master)](https://travis-ci.org/hmrc/component-finder)
 
-### Requirements
+```
+$ node index.js foo
```

Comment:
That's changed as part of that PR https://github.com/hmrc/component-finder/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8L26